qubes/vm/qubesvm.py

```diff
@@ -660,6 +713,24 @@ class QubesVM(qubes.vm.mix.net.NetVMMixin, qubes.vm.BaseVM):
     #
     # properties loaded from XML
     #
+    active_bootmode = qubes.property(
+        "active_bootmode",
+        type=str,
+        load_stage=4,
+        default=_default_active_bootmode,
+        doc="Active boot mode for this domain"
+    )
+
+    bootmode_kernelopts = qubes.property(
```

Comment:
This doesn't need to be visible in qvm-prefs, no? You can make it a normal property (`@property` decorator). But if you want this to be visible in qvm-prefs as a read-only property anyway, use `@qubes.stateless_property` decorator. You have examples of both below.

Reply:
Sounds good, I'll port it to `@property`. (I did like having it show up in qvm-prefs a bit, but it's not at all necessary, *and* it has an annoying space in the front that the libvirt template depends on so it looks bad in qvm-prefs' output anyway.)

Reply:
Switched to `@property`.